blueflood-core/src/test/java/com/rackspacecloud/blueflood/io/serializers/json/IMetricSerializerTest.java

```diff
@@ -1,138 +0,0 @@
-/*
- * Copyright 2014 Rackspace
- *
- *    Licensed under the Apache License, Version 2.0 (the "License");
- *    you may not use this file except in compliance with the License.
- *    You may obtain a copy of the License at
- *
- *        http://www.apache.org/licenses/LICENSE-2.0
- *
- *    Unless required by applicable law or agreed to in writing, software
- *    distributed under the License is distributed on an "AS IS" BASIS,
- *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *    See the License for the specific language governing permissions and
- *    limitations under the License.
- */
-
-package com.rackspacecloud.blueflood.io.serializers.json;
-
-import com.rackspacecloud.blueflood.io.serializers.json.IMetricSerializer;
-import com.rackspacecloud.blueflood.types.*;
-import com.rackspacecloud.blueflood.utils.TimeValue;
-import junit.framework.Assert;
-import org.codehaus.jackson.map.ObjectMapper;
-import org.junit.Test;
-
-import java.io.IOException;
-import java.math.BigInteger;
-import java.util.concurrent.TimeUnit;
-
-public class IMetricSerializerTest {
```

Comment:
This class tested the com.rackspacecloud.blueflood.io.serializers.json.* classes, which I removed as they were no longer used in the codebase.